pennylane/workflow/qnode.py

```diff
@@ -14,7 +14,7 @@
 """
 This module contains the QNode class and qnode decorator.
 """
-# pylint: disable=too-many-instance-attributes,too-many-arguments,protected-access,unnecessary-lambda-assignment, too-many-branches, too-many-statements, unused-argument, too-many-positional-arguments, inconsistent-return-statements
+# pylint: disable=too-many-instance-attributes,too-many-arguments,protected-access,unnecessary-lambda-assignment, too-many-branches, too-many-statements, unused-argument, too-many-positional-arguments, inconsistent-return-statements, redefined-outer-name

```

Comment:
What's the outer name we are redefining?

Reply:
Ah, looks like nothing. Probably from an old commit. I cleaned up all of the pylints so now they should all be covering something.

Reply:
Will be a couple updates in #6621  anyway.